.travis.yml

```
@@ -55,7 +55,10 @@ matrix:
   - env: JDK="jdk11" GATE="build,bootstraplite" PRIMARY="compiler"
   - os: linux
     arch: arm64
-    env: JDK="jdk11" GATE="fullbuild,test" PRIMARY="compiler"
+    env: JDK="jdk11" GATE="fullbuild,test:1:100" PRIMARY="compiler"
```

This should be
```suggestion
    env: JDK="jdk11" GATE="fullbuild,test:1" PRIMARY="compiler"
```
but depends on https://github.com/graalvm/mx/pull/222